engine/api/websocket.go

```diff
@@ -302,6 +299,18 @@ func (c *websocketClient) updateEventFilters(ctx context.Context, db gorp.SqlExe
 			return err
 		}
 		switch f.Type {
+		case sdk.WebsocketFilterTypeDryRunRetentionWorkflow:
```

Comment:
Can be grouped with case sdk.WebsocketFilterTypeWorkflow, sdk.WebsocketFilterTypeAscodeEvent: